### PR TITLE
🌐  Add Dutch translation for `docs/nl/docs/features.md`

### PR DESCRIPTION
**Pull request by maxscheijen**
Added Dutch translation to docs/nl/docs/features.md

### CODE REVIEW

File: docs/nl/docs/features.md [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+}
+
+my_second_user: User = User(**second_user_data)
+```
+
+/// info
+
+`**second_user_data` betekend:
+
+Geef de keys en values van de `second_user_data` dict direct door als key-value argumenten, gelijk aan: `User(id=4, name=“Mary”, joined=“2018-11-30”)`
```
Comment (svlandeg) [inline docs/nl/docs/features.md:70]: "keys", "values" & "dict" still need to be translated. Perhaps to "sleutels", "waarden" & "mapping"?

File: docs/nl/docs/features.md [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+
+* in <a href="https://code.visualstudio.com/" class="external-link" target="_blank">Visual Studio Code</a>:
+
+![editor ondersteuning](https://fastapi.tiangolo.com/img/vscode-completion.png)
+
+* in <a href="https://www.jetbrains.com/pycharm/" class="external-link" target="_blank">PyCharm</a>:
+
+![editor ondersteuning](https://fastapi.tiangolo.com/img/pycharm-completion.png)
+
+Je krijgt autocomletion die je voorheen misschien zelfs voor onmogelijk had gehouden. Zoals bijvoorbeeld de `price` key in een JSON body (die genest had kunnen zijn) die afkomstig is van een request.
```
Comment (svlandeg) [inline docs/nl/docs/features.md:94]: "autocompletion" has been translated to "(automatische) code aanvulling" in #12042

File: docs/nl/docs/features.md [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+    * URL.
+    * E-mail.
+    * UUID.
+    * ...en anderen.
+
+Alle validatie wordt uitgevoerd door het beproefde en robuuste **Pydantic**.
+
+### Beveiliging en authenticatie
+
+Beveiliging en authenticatie is geïntegreerd. Zonder compromissen te doen naar databases of datasmodellen.
```
Comment (svlandeg) [inline docs/nl/docs/features.md:122]:
```suggestion
Beveiliging en authenticatie is geïntegreerd. Zonder compromissen te doen naar databanken of datamodellen.
```

File: docs/nl/docs/features.md [hunk 4]
```diff
@@ -0,0 +1,201 @@
...
+* HTTP Basic.
+* **OAuth2** (ook met **JWT tokens**). Bekijk de tutorial over [OAuth2 with JWT](tutorial/security/oauth2-jwt.md){.internal-link target=_blank}.
+* API keys in:
+    * Headers.
+    * Query parameters.
+    * Cookies, etc.
+
+Plus alle beveiligingsfuncties van Starlette (inclusief **sessiecookies**).
+
+Gebouwd als een herbruikbare tool met componenten die makkelijk zijn te integreren in en met je systemen, datastores, relationele en NoSQL databases, etc.
```
Comment (svlandeg) [inline docs/nl/docs/features.md:135]:
```suggestion
Gebouwd als een herbruikbare tool met componenten die makkelijk te integreren zijn in en met je systemen, datastores, relationele en NoSQL databanken, enz.
```

File: docs/nl/docs/features.md [hunk 5]
```diff
@@ -0,0 +1,201 @@
...
+### Dependency Injection
+
+FastAPI bevat een uiterst eenvoudig, maar uiterst krachtig <abbr title='ook bekend als "components", "resources", "services", "providers"'><strong>Dependency Injection</strong></abbr> systeem.
+
+* Zelfs dependencies kunnen dependencies hebben, waardoor een hiërarchie of **“graph” van dependencies** ontstaat.
+* Allemaal **automatisch afgehandeld** door het framework.
+* Alle dependencies kunnen data nodig hebben van request, de vereiste **padoperaties veranderen** en automatische documentatie verstrekken.
+* **Automatische validatie** zelfs voor *padoperatie* parameters gedefinieerd in dependencies.
+* Ondersteuning voor complexe gebruikersauthenticatiesystemen, **databaseverbindingen**, etc.
+* **Geen compromis** met databases, frontends, etc. Maar eenvoudige integratie met ze allemaal.
```
Comment (svlandeg) [inline docs/nl/docs/features.md:146]:
```suggestion
* **Geen compromisen** met databanken, gebruikersinterfaces, enz. Maar eenvoudige integratie met ze allemaal.
```

File: docs/nl/docs/features.md [hunk 6]
```diff
@@ -0,0 +1,201 @@
...
+* **CORS**, GZip, Statische bestanden, Streaming reacties.
+* **Sessie en Cookie** ondersteuning.
+* 100% van de code is getest.
+* 100% type geannoteerde codebase.
+
+## Pydantic functionaliteit
+
+**FastAPI** is volledig verenigbaar met (en gebaseerd op) Pydantic. Dus alle extra <a href="https://docs.pydantic.dev/" class="external-link" target="_blank"><strong>Pydantic</strong></a> code die je nog hebt werkt ook.
+
+Inclusief externe pakketten die ook gebaseerd zijn op Pydantic, zoals <abbr title="Object-Relational Mapper">ORM</abbr>s, <abbr title="Object-Document Mapper">ODM</abbr>s voor databases.
```
Comment (svlandeg) [inline docs/nl/docs/features.md:182]:
```suggestion
Inclusief externe pakketten die ook gebaseerd zijn op Pydantic, zoals <abbr title="Object-Relational Mapper">ORM</abbr>s, <abbr title="Object-Document Mapper">ODM</abbr>s voor databanken.
```

File: docs/nl/docs/features.md [hunk 7]
```diff
@@ -0,0 +1,201 @@
...
+    * Query parameters.
+    * Cookies, etc.
+
+Plus alle beveiligingsfuncties van Starlette (inclusief **sessiecookies**).
+
+Gebouwd als een herbruikbare tool met componenten die makkelijk zijn te integreren in en met je systemen, datastores, relationele en NoSQL databases, etc.
+
+### Dependency Injection
+
+FastAPI bevat een uiterst eenvoudig, maar uiterst krachtig <abbr title='ook bekend als "components", "resources", "services", "providers"'><strong>Dependency Injection</strong></abbr> systeem.
```
Comment (svlandeg) [inline docs/nl/docs/features.md:139]: Can we translate none of these "components", "resources", "services", "providers"?

File: docs/nl/docs/features.md [hunk 8]
```diff
@@ -0,0 +1,201 @@
...
+* 100% van de code is getest.
+* 100% type geannoteerde codebase.
+
+## Pydantic functionaliteit
+
+**FastAPI** is volledig verenigbaar met (en gebaseerd op) Pydantic. Dus alle extra <a href="https://docs.pydantic.dev/" class="external-link" target="_blank"><strong>Pydantic</strong></a> code die je nog hebt werkt ook.
+
+Inclusief externe pakketten die ook gebaseerd zijn op Pydantic, zoals <abbr title="Object-Relational Mapper">ORM</abbr>s, <abbr title="Object-Document Mapper">ODM</abbr>s voor databases.
+
+Dit betekent ook dat je in veel gevallen het object dat je van een request krijgt **direct naar je database** kunt sturen, omdat alles automatisch wordt gevalideerd.
```
Comment (svlandeg) [inline docs/nl/docs/features.md:184]:
```suggestion
Dit betekent ook dat je in veel gevallen het object dat je van een request krijgt **direct naar je databank** kunt sturen, omdat alles automatisch wordt gevalideerd.
```

File: docs/nl/docs/features.md [hunk 9]
```diff
@@ -0,0 +1,201 @@
...
+
+Hetzelfde geldt ook andersom, in veel gevallen kun je dus het object dat je krijgt van de database **direct** doorgeven aan de client**.
```
Comment (svlandeg) [inline docs/nl/docs/features.md:186]:
```suggestion
Hetzelfde geldt ook andersom, in veel gevallen kun je dus het object dat je krijgt van de databank **direct doorgeven aan de client**.
```

File: docs/nl/docs/features.md [hunk 10]
```diff
@@ -0,0 +1,201 @@
...
+
+### Dependency Injection
+
+FastAPI bevat een uiterst eenvoudig, maar uiterst krachtig <abbr title='ook bekend als "components", "resources", "services", "providers"'><strong>Dependency Injection</strong></abbr> systeem.
+
+* Zelfs dependencies kunnen dependencies hebben, waardoor een hiërarchie of **“graph” van dependencies** ontstaat.
+* Allemaal **automatisch afgehandeld** door het framework.
+* Alle dependencies kunnen data nodig hebben van request, de vereiste **padoperaties veranderen** en automatische documentatie verstrekken.
+* **Automatische validatie** zelfs voor *padoperatie* parameters gedefinieerd in dependencies.
+* Ondersteuning voor complexe gebruikersauthenticatiesystemen, **databaseverbindingen**, etc.
```
Comment (svlandeg) [inline docs/nl/docs/features.md:145]:
```suggestion
* Ondersteuning voor complexe gebruikersauthenticatiesystemen, **databankverbindingen**, etc.
```

File: docs/nl/docs/features.md [hunk 11]
```diff
@@ -0,0 +1,201 @@
...
+
+Hetzelfde geldt ook andersom, in veel gevallen kun je dus het object dat je krijgt van de database **direct** doorgeven aan de client**.
+
+Met **FastAPI** krijg je alle functionaliteit van **Pydantic** (omdat FastAPI is gebaseerd op Pydantic voor alle dataverwerking):
+
+* **Geen brainfucks**:
+    * Je hoeft geen nieuwe microtaal voor schemadefinities te leren.
+    * Als je bekend bent Python types, weet je hoe je Pydantic moet gebruiken.
+* Werkt goed samen met je **<abbr title=“Integrated Development Environment, vergelijkbaar met een code editor”>IDE</abbr>/<abbr title=“Een programma dat controleert op fouten in de code”>linter</abbr>/hersenen**:
+    * Doordat pydantic's data structuren enkel instanties zijn van classes, die je definieert, werken autocompletion, linting, mypy en je intuïtie allemaal goed met je gevalideerde data.
```
Comment (svlandeg) [inline docs/nl/docs/features.md:194]:
```suggestion
    * Doordat pydantic's datastructuren enkel instanties zijn van klassen, die je definieert, werkt automatische aanvulling, linting, mypy en je intuïtie allemaal goed met je gevalideerde data.
```

File: docs/nl/docs/features.md [hunk 12]
```diff
@@ -0,0 +1,201 @@
...
+* Valideer **complexes structuren**:
```
Comment (svlandeg) [inline docs/nl/docs/features.md:195]:
```suggestion
* Valideer **complexe structuren**:
```

File: docs/nl/docs/features.md [hunk 13]
```diff
@@ -0,0 +1,201 @@
...
+    * Gebruik van hiërarchische Pydantic modellen, Python `typing`'s `List` en `Dict`, etc.
+    * Met validators kunnen complexe dataschema's duidelijk en eenvoudig worden gedefinieerd, gecontroleerd en gedocumenteerd als JSON Schema.
+    * Je kunt diep geneste JSON** objecten laten valideren en annoteren.
```
Comment (svlandeg) [inline docs/nl/docs/features.md:198]:
```suggestion
    * Je kunt diep **geneste JSON** objecten laten valideren en annoteren.
```

**Review by svlandeg — COMMENTED** [review]
Thanks so much for this, super appreciated! 🙏 

This mostly looks good - I confirmed that all the English sections are properly translated and up-to-date with this PR. I left some detailed review comments to address typo's and suggest a few additional translations, as well as ensuring consistency with the previous Dutch PR https://github.com/fastapi/fastapi/pull/12042 (also mostly by you @maxscheijen, so thanks again for that!)

File: docs/nl/docs/features.md (hunk 1 above)
Comment (maxscheijen) [reply docs/nl/docs/features.md:70]: I found this a difficult one. Every Dutch speaking python programmer I know talks about "keys" and "values" in the context of a dictionary. In my view, FastAPI is aimed at people that have basic understanding of python programming. Renaming them to "sleutels" and "waarden" would, in my opinion, create unnecessary confusion.

Maybe use the Dutch words but add the English transaction after it in parentheses (or the other way around)?

```markdown
Geef de sleutels (keys) en waarden  values van de `second_user_data` dict direct door als sleutel-waarden argumenten, gelijk aan: `User(id=4, name=“Mary”, joined=“2018-11-30”)`
```

File: docs/nl/docs/features.md (hunk 3 above)
Comment (maxscheijen) [reply docs/nl/docs/features.md:122]: Isn't database also a Dutch word? I think it's in the dictionary.

File: docs/nl/docs/features.md (hunk 1 above)
Comment (svlandeg) [reply docs/nl/docs/features.md:70]: Yea, I get what you mean. I honestly find some of these translations a bit awkward myself. But your solution sounds good to me, so something like "sleutels (keys) en waarden (values)" should work.

File: docs/nl/docs/features.md (hunk 3 above)
Comment (svlandeg) [reply docs/nl/docs/features.md:122]: Maybe, Dutch has a lot of English "loan words", but I think "databank" is a very clear Dutch word that is used a lot in Dutch text books and other literature, so I would definitely prefer it over "database".

Comment (maxscheijen) [reply docs/nl/docs/features.md:122]: I think you are right. The Dutch spoken in The Netherlands uses more English words in comparison to the Dutch spoken in for example Belgium.

 At universities in The Netherlands, Dutch spoken computer science (informatica) programs use "databases".

- https://www.uu.nl/bachelors/informatica/studieprogramma
- https://www.uva.nl/programmas/bachelors/informatica/studieprogramma/studieprogramma.html

> e.g. first site I look, informatica opleiding at UGent, has a course called "databanken": https://informatica.ugent.be/bachelor/

Yes that's indeed the case. However, "Databases" is also the name of several courses at Belgian universities.

- https://www.uantwerpen.be/nl/studeren/aanbod/alle-opleidingen/informatica-studeren/bachelor/studieprogramma/
- https://www.uhasselt.be/nl/studeren/opleidingen/bachelor-en-master-in-de-informatica#anch-215-bachelor

The most recent edition of the Dutch book ""Principes van databanken" by Guy de Tré that is used in the UGent course is published with the title "Principes van databases".

The search term for "database" is also more popular than "databank" in both Belgium and The Netherlands.

- https://trends.google.com/trends/explore?date=all&geo=BE&q=database,databank&hl=nl-NL
- https://trends.google.com/trends/explore?date=all&geo=NL&q=database,databank&hl=nl-NL

So I don't really think there is a correct definitive correct Dutch translation of "databases".

However, in my view, the term “database” is more often used in Dutch (in Belgium and The Netherlands, in Universities, in text books and in government).

I think "database" leads to less confusion for the user when also taking into account that they are probably also reading other (English) documentation.

Comment (svlandeg) [reply docs/nl/docs/features.md:122]: > So I don't really think there is a correct definitive correct Dutch translation of "databases".

Haha - I respectfully disagree: the clear correct Dutch translation of "database" is "databank", but the question is whether the actual Dutch translation, or the English loan word, is used more frequently in Dutch texts 🤷 

Anyway, why don't we adhere to this rule of thumb in cases like these: to follow the Dutch-speaking Wikipedia? While that's usually also a bit skewed towards The Netherlands vs Belgium, at least it's an "authorative" source to use as gold truth.

In this particular case: that would mean we stick to "database" as https://nl.wikipedia.org/wiki/Databank redirects to https://nl.wikipedia.org/wiki/Database.

Please do go through my reviewer's comments carefully though - some of the "databank" suggestions also contain fixes for other words 😉

File: docs/nl/docs/features.md [hunk 14]
```diff
@@ -0,0 +1,201 @@
...
+}
+
+my_second_user: User = User(**second_user_data)
+```
+
+/// info
+
+`**second_user_data` betekent:
+
+Geef de sleutels (keys) en waarden (values) de `second_user_data` dict direct door als sleutel-waarden argumenten, gelijk aan: `User(id=4, name=“Mary”, joined=“2018-11-30”)`
```
Comment (svlandeg) [inline docs/nl/docs/features.md:70]:
```suggestion
Geef de sleutels (keys) en waarden (values) van de `second_user_data` dict direct door als sleutel-waarden argumenten, gelijk aan: `User(id=4, name=“Mary”, joined=“2018-11-30”)`
```

**Review by svlandeg — APPROVED** [review]
Nice work! 🎉

**Comment by alejsdev** [comment]
Thanks for the help! @maxscheijen @svlandeg :sparkles:

**Pull request merged**